terraform/eks.tf

```diff
@@ -48,14 +48,17 @@ resource "null_resource" "eks-bootstrap" {
     command = <<EOF
     kubectl patch configmap/aws-auth --patch "${local.aws_auth_configmap_yaml}" -n kube-system --kubeconfig <(echo $KUBECONFIG | base64 --decode);
     kubectl set env daemonset aws-node -n kube-system ENABLE_PREFIX_DELEGATION=true --kubeconfig <(echo $KUBECONFIG | base64 --decode);
-    kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "docker-pull-secret"}]}' --kubeconfig <(echo $KUBECONFIG | base64 --decode)
+    kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "docker-pull-secret"}]}' --kubeconfig <(echo $KUBECONFIG | base64 --decode);
+    kubectl patch serviceaccount default -n kube-system -p '{"imagePullSecrets": [{"name": "docker-pull-secret"}]}' --kubeconfig <(echo $KUBECONFIG | base64 --decode)
```

Comment:
Why did we have to do 2 commands here to set up the docker pull secret?  (`-n` and `-p`)

Are they for diff namespaces

Reply:
Yep, the second command still has the `-p`, it's just after the `-n <namespace>`

Reply:
Ohh wait got it